packages/dev/core/src/Misc/tools.ts

```diff
@@ -183,20 +183,14 @@ export class Tools {
         return count === value;
     }
 
-    private static _TmpFloatArray = new Float32Array(1);
-
     /**
      * Returns the nearest 32-bit single precision float representation of a Number
      * @param value A Number.  If the parameter is of a different type, it will get converted
      * to a number or to NaN if it cannot be converted
      * @returns number
      */
     public static FloatRound(value: number): number {
-        if (Math.fround) {
-            return Math.fround(value);
-        }
-
-        return (Tools._TmpFloatArray[0] = value), Tools._TmpFloatArray[0];
+        return Math.fround(value);

```

Comment:
I wonder why we haven't done that earlier

Reply:
Because it was not supported by IE11, and when we stopped supporting this browser, nobody remembered that this code could be changed :) (there was no comment saying it was a workaround because of IE11).